### PR TITLE
Make clear that update is in progress

### PR DESCRIPTION
**Pull request by cgreene**
There has been a fair bit of twitter traffic about the paper. This updates the title to try to make clear that the update is in progress and not yet finalized. Hopefully this will help readers understand this.

### CODE REVIEW

**Comment by agitter** [comment]
Should we drop the "2019" from the title?  If we do release a new version, it definitely will not be this month!

**Comment by stephenra** [comment]
@agitter Agree that "2019" ought to be dropped.

**Comment by agitter** [comment]
We might as well drop 2019 from the header as well:
> An effort is underway to update the manuscript ~to capture the state of the field in 2019~.

**Comment by evancofer** [comment]
Since there are only a few more weeks in the year, I agree that 2019 should probably be dropped.

File: content/00.front-matter.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ##}
 
 _A published version of this manuscript from 04 April 2018 is available at <https://doi.org/10.1098/rsif.2017.0387>.
-An effort is underway to update the manuscript to capture the state of the field in 2019.
```
Comment (agitter) [inline content/00.front-matter.md]: @cgreene is it okay with you if I remove this part as well?

Comment (cgreene) [reply content/00.front-matter.md]: Yes - thanks!

**Pull request merged**